test/e2e/modal.test.js

```diff
@@ -105,5 +106,27 @@ describe('vl-modal', async () => {
         await vlModalPage.openModalSafari();
         const content = await modal.getContent();
         await assert.eventually.isTrue(content.hasFocus());
+        await modal.close();
+    });
+
+    it('als gebruiker zie ik een verticale scrollbar als er te veel tekst in de modal staat', async() => {
+        await vlModalPage.openModalMetVeelTekst();
+        const modal = await vlModalPage.getModalMetVeelTekst();
+        await assert.eventually.isTrue(driver.executeScript("return arguments[0].scrollHeight > arguments[0].clientHeight;", await modal._getDialog()));
+        await modal.cancel();
+    });
+
+    it('als gebruiker zie ik geen (disabled) scrollbar als de modal groot genoeg is voor de tekst', async () => {
+        await vlModalPage.openModalZonderButtonEnContent();
+        const modal = await vlModalPage.getModalZonderButtonEnContent();
+        await assert.eventually.isTrue(driver.executeScript("return arguments[0].offsetWidth == arguments[0].clientWidth", await modal._getDialog()));
+        await modal.cancel();
+    });
```

Comment:
Ik ga nog eens kijken of dit ook kan met de Selenium API zodat er minder JS logica nodig is en het meer gericht is op de gebruiker.

Reply:
Ik heb de test aangepast naar:
```JavaScript
it('als gebruiker zie ik een verticale scrollbar als er te veel tekst in de modal staat', async() => {
    await vlModalPage.openModalMetVeelTekst();
    const modal = await vlModalPage.getModalMetVeelTekst();
    const dialog = await modal._getDialog();
    await driver.executeScript("return arguments[0].scrollTop = 0;", dialog);
    await modal.cancel();
});
```

Deze test was initieel rood met volgende foutmelding: `element not interactable`op de cancel.